src/renderer/components/commands-editors.tsx

```diff
@@ -82,6 +82,14 @@ export class EditorDropdown extends React.Component<EditorDropdownProps, EditorD
       );
     }
 
+    result.push(
+      <Button
+        icon='grid-view'
+        text='Reset Layout'
+        onClick={appState.resetEditorLayout}
+      />
+    );
+
```

Comment:
Once all of the Editors have been set in the Array, I thought it made sense to then push the Reset Layout button at the end. 

This made more sense to me as a user, but please shout if you disagree!